pkg/cmd/cli/describe/describer.go

```diff
@@ -336,3 +341,70 @@ func (d *PolicyBindingDescriber) Describe(namespace, name string) (string, error
 		return nil
 	})
 }
+
+// TemplateDescriber generates information about a template
+type TemplateDescriber struct {
+	client.Interface
+	meta.MetadataAccessor
+	runtime.ObjectTyper
+	DescribeObject func(obj runtime.Object, out *tabwriter.Writer) (bool, error)
+}
+
+func (d *TemplateDescriber) DescribeParameters(params []templateapi.Parameter, out *tabwriter.Writer) {
+	for i, p := range params {
+		formatString(out, fmt.Sprintf("Parameter %d\tName", i+1), p.Name)
+		formatString(out, "\tDescription", p.Description)
+		if len(p.Generate) > 0 {
```

Comment:
shouldn't this be 
len(p.Generate) <= 0 {
  formatString(out, "\tValue", p.Value)
  continue
}
other code

to reduce indentation and the else clause? :)